modules/aurora_database_cluster/variables.tf

```
@@ -0,0 +1,105 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+variable "network_id" {
+  description = "The identity of the VPC in which the security group attached to the PostgreSQL RDS instance will be deployed."
+  type        = string
+}
+
+variable "db_name" {
+  type        = string
+  description = "PostgreSQL instance name. No special characters."
+}
+
+variable "db_username" {
+  type        = string
+  description = "PostgreSQL instance username. No special characters."
+}
```

🙏 Lets please use the validation feature if we have such requirements.
```suggestion
variable "db_name" {
  type        = string
  description = "PostgreSQL instance name. No special characters."
  
    validation {
    condition     = can(regex("^[a-zA-Z0-9]+$", var.db_name))
    error_message = "The db_name must only contain alphanumeric characters."
  }
}

variable "db_username" {
  type        = string
  description = "PostgreSQL instance username. No special characters."
  
  validation {
    condition     = can(regex("^[a-zA-Z0-9]+$", var.db_username))
    error_message = "The db_name must only contain alphanumeric characters."
  }
}
```